Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/J2ObjCParser.kt

```diff
@@ -60,9 +64,31 @@ class J2ObjCParser: ClassVisitor(Opcodes.ASM5) {
       isForwardDeclaration = false,
       location = Location(headerId = HeaderId("usr/include/objc/NSObject.h")) // TODO: When implementing inheritance check for proper base class.
     )
+    generatedClass.protocols.addAll(interfaceNames.map{ObjCProtocolImpl(
+      name = it,
+      isForwardDeclaration = false,
```

Comment:
Isn't this explicitly a forward declaration.

Reply:
Yep, done.